### ISSUE TITLE
when updating avatar images, local cached avatar image is still rendered

### ISSUE THREAD

**Issue by McPizza0** [uninbox/UnInbox#32]
when a user updates their profile or org profile avatar, the browser still uses the locally cached image
this is because avatar URLs are fixed and do not change with updates

we have a workaround in place to append a timestamp to the avatar URLs, but this means that the app has to re-fetch the avatar from the server every single time, incurring latency and cost ($) for transport
https://github.com/uninbox/UnInbox/blob/904fb08d27f45b2555cead718ea06b36cc7784b9/apps/web-app/composables/utils.ts#L32

we need a better permanent fix for this issue

possible solution:
- add `updatedAt` column to `userProfile`, `orgs`, `userGroups` & `contacts` tables. append the `updatedAt` value to `generateAvatarUrl` function

thoughts @cstrnt ?

**Comment by skushagra9** [comment]
can I work on this ??

**Comment by Praashh** [comment]
if @skushagra9 you need any help, can ping me.

**Comment by skushagra9** [comment]
ok

**Comment by McPizza0** [comment]
lets wait for @cstrnt 's input before starting

**Comment by skushagra9** [comment]
okay

**Comment by cstrnt** [comment]
#33 is kinda related to that, I've left a comment there on caching

**Comment by McPizza0** [comment]
closing in favor of implementation in #33
